src/chef_authn.erl

```diff
@@ -510,10 +563,18 @@ verify_sig(Plain, BodyHash, ContentHash, AuthSig, UserId, PublicKey, SignVersion
     %% if the content hash is wrong.
     ContentHash = BodyHash,
     {name, UserId};
-verify_sig(Plain, _BodyHash, _ContentHash, AuthSig, UserId, PublicKey, ?SIGNING_VERSION_V1_2) ->
+verify_sig(Plain, _BodyHash, _ContentHash, AuthSig, UserId, PublicKey, {_, ?SIGNING_VERSION_V1_2}) ->
     true = public_key:verify(Plain, sha, base64:decode(AuthSig), decode_key_data(PublicKey)),
+    {name, UserId};
+verify_sig(Plain, _BodyHash, _ContentHash, AuthSig, UserId, PublicKey,
+           {SignAlgorithm, ?SIGNING_VERSION_V1_3}) ->
+    true = public_key:verify(Plain, openssl_algorithm_name(SignAlgorithm),
```

Comment:
the header files indicate that this is true
https://github.com/openssl/openssl/blob/master/include/openssl/rsa.h#L212-L220